src/typechecker/define-class.lisp

```diff
@@ -334,7 +334,9 @@
            :if (not (zerop class-arity))
              :do (setf env (tc:set-function env codegen-sym (tc:make-function-env-entry
                                                              :name codegen-sym
-                                                             :arity class-arity)))
+                                                             :arity class-arity
+                                                             ;; FIXME: Under what circumstances would we want fundep to be inline?
```

Comment:
Class constructor functions can't be inlined by the Coalton inliner because they don't appear in the code env.